File: OP_PROJECT_JAVA/src/game/Animals/Wolf.java
package game.Animals;

import game.Animal;
import game.Position;
import game.World;

public class Wolf extends Animal {
    private static final int WOLF_STRENGTH = 9;
    private static final int WOLF_INIT = 5;
    private static final String WOLF_TYPE = "Wolf";


    public Wolf(Position position, World world) {
        super(WOLF_TYPE, WOLF_STRENGTH, WOLF_INIT, position, world);
    }
}

File: OP_PROJECT_PYTHON/src/game/Animals/Wolf.py
from src.game.Animal import Animal


class Wolf(Animal):
    WOLF_STRENGTH = 9
    WOLF_INIT = 5
    WOLF_TYPE = "Wolf"

    def __init__(self, position, world):
        super().__init__(Wolf.WOLF_TYPE, Wolf.WOLF_STRENGTH, Wolf.WOLF_INIT, position, world)
